packages/widgetbook/lib/src/navigation/models/widgetbook_use_case_data.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/cupertino.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_core/widgetbook_core.dart';
+
+@immutable
+class WidgetbookUseCaseData extends Model {
+  const WidgetbookUseCaseData({
+    required this.path,
+    required this.builder,
+  }) : super(id: path);
+
+  final String path;
+  final UseCaseBuilder builder;
+
+  @override
+  bool operator ==(Object other) {
+    if (identical(this, other)) return true;
+
+    return other is WidgetbookUseCaseData &&
+        other.path == path &&
+        other.builder == builder;
+  }
+
+  @override
+  int get hashCode => builder.hashCode;
+}
```

Comment:
Since `Model` is not a freezed class, I think we can make this one a freezed class